### ISSUE TITLE
Invalid content-type header causes LookupError

### ISSUE THREAD

**Issue by elektito** [aio-libs/aiohttp#2120]
## Long story short

I've encountered a website that sends out an invalid Content-Type header. They send `text/xml; charset=utf-8,charset=utf-8`, which apart from having two charset parameters, uses a comma as a separator instead of a semicolon, causing charset to be set to "utf-8,charset=utf-8", but that's besides the point I guess.

The point is charset has an invalid value and when `text()` is called, the decoding attempt causes a `LookupError` to be raised.

## Expected behaviour

I'm not sure, really. Is this the expected behavior? It sounds a bit ugly. Perhaps a clearer exception should be used.

## Actual behaviour

As described above, a LookupError, like this:

    ~/source/forcefeed/venv/lib/python3.5/site-packages/aiohttp/client_reqrep.py in text(self, encoding, errors)
        711             encoding = self._get_encoding()
        712
    --> 713         return self._content.decode(encoding, errors=errors)
        714
        715     @asyncio.coroutine
    
    LookupError: unknown encoding: utf-8,charset=utf-8

## Steps to reproduce

    session = aiohttp.ClientSession()
    async with session.get('http://farzand.net/feed/rss') as r:
        print(await r.text())

## Your environment

I have Python 3.5.2, and use aiohttp 2.2.3.

**Comment by asvetlov** [comment]
aiohttp obviously cannot fix current behavior, you should use `resp.read()` for getting binary  stream and encode it explicitly.
Adding new exception type doesn't improve situation too much.
Exception message is pretty clean, isn't it?

**Comment by elektito** [comment]
Yes, the message is exceedingly clear! I just thought leaking the underlying exception might not be the intended behavior.

IMHO, adding a new exception helps developers write more robust code. In the current situation, I can't be sure what exceptions an aiohttp functions might raise. I started handling all ClientErrors, until I realized there are other exceptions raised from the underlying layers, like CertificateError, TimeoutError, UnicodeDecodeError, LookupError and more

This way I can never be sure if I'm handling all error situations or not.
